class-00/lab-b/lab/app.js

```diff
@@ -0,0 +1,198 @@
+// We no longer need the curly braces and the return is implicit
+// Without an arrow function, we need to explicitly type "return"
+const oneLiner = course => `I cam currently enrolled in ${course}`;
+
+// TODO: Uncomment the following line of code to see the output in the browser console
+console.log('As a one-liner:', oneLiner('Code 301'));
+
+
+// STEP 5
+// What if we have multiple parameters?
+// In a function expression, they all go in the parentheses
+const add = function(num1, num2) {
+  return `${num1} + ${num2} = ${num1 + num2}`;
+};
+
+// TODO: Uncomment the following line of code to see the output in the browser console
+console.log('Let\'s do some math:', add(4, 5));
+
+
+// STEP 6
+// When we have multiple parameters, they must be wrapped in parentheses
+// We can only remove the parentheses when there is a single parameter
+const addOneLiner = (num1, num2) => `${num1} + ${num2} = ${num1 + num2}`;
+
+// TODO: Uncomment the following line of code to see the output in the browser console
+console.log('Add as a one-liner:', addOneLiner(4, 5));
+
+
+// STEP 7
+// What if we have multiple lines of code?
+// We need to use a code block
+const multiLiner = word => {
+  word = word.toUpperCase();
+  return word;
+};
+
+// TODO: Uncomment the following line of code to see the output in the browser console
+console.log('Multi-line arrow function:', multiLiner('hello'));
+
+
+// STEP 8
+// The way an object is returned is different with an arrow function, too.
+// Here is how we return an object without arrow functions
+const oldObject = function(array) {
+  return {
+    firstValue: array[0],
+    secondValue: array[1],
+    thirdValue: array[2]
+  };
+};
+
+// TODO: Uncomment the following line of code to see the output in the browser console
+console.log('Hello from the old object function', oldObject(['hi', 'hello', 'are you there?']));
+
+
+// STEP 9
+// With an arrow function, we need to wrap our object in parentheses
+// Otherwise, it will be interpreted as a code block
+const newObject = array => ({
+  firstValue: array[0],
+  secondValue: array[1],
+  thirdValue: array[2]
+});
+
+// TODO: Uncomment the following line of code to see the output in the browser console
+console.log('Hello from the new object function', newObject(['hi', 'hello', 'are you there?']));
+
+
+// STEP 10
+// Uncomment the console.log lines to view the output of each function in the browser console.
+// Refactor each function into an arrow function.
+// Write your solutions on a single line wherever possible.
+
+let sum = (a, b, c, d) => a + b + c + d;
+
+
+// TODO: Uncomment the following line of code to see the output in the browser console
+console.log(sum(1, 2, 3, 4));
+
+
+let objectLit = () => ({
+    key1: 'value1',
+    key2: 'value2',
+    key3: 'value3',
+});
+
+// TODO: Uncomment the following line of code to see the output in the browser console
+console.log(objectLit());
+
+
+let sumAndProduct = (a, b) => {
```

Comment:
This can be shortened further with arrow function format:

`let sumAndProduct = (a, b) => [a + b, a * b];`